### DISCUSSION TITLE
Trigger teplaters one by one

### DISCUSSION THREAD

**Discussion by Gr-Nobel** [SilentVoid13/Templater#935] [Help]
I have 5 tp.suggester in one page. Is there any way to trigger them one by one. the command palate has only one option to replace all template files.

**Comment by Zachatoo** [comment]
I do not believe there is a way to do that. What's the use case?

**Comment by Gr-Nobel** [comment]
I have bunch of multi choice templates on a single page(my daily note).
I want to execute them on different times during the day.

**Reply by AB1908** [reply]
Hmm more likely than not, you'd have to split them off into different templates. However, you can use moment() to check what time of day it is and use some JS conditional logic to choose the right template type.
